api/swagger.yml

```diff
@@ -2831,7 +2831,21 @@ paths:
       responses:
         204:
           description: NoContent
-
+  /config/version:
+    get:
+      tags:
+        - config
+      operationId: getLakeFSVersion
+      description: get version of lakeFS server
+      responses:
+        200:
+          description: lakeFS storage configuration
+          content:
+            application/json:
+              schema:
+                type: string
```

Comment:
This should be an object.  That way we can add more details, e.g. API versions, plugin versions, etc.

Passing a primitive or an array as the entire JSON response is somewhat similar to "primitive obsession", and can be particularly nasty in APIs.